text/accumulation.tex

```diff
@@ -30,118 +35,207 @@ \subsection{Preimage Integration}
+  (\chi', \delta^\ddagger, \iota', \varphi') &\equiv \mathbf{o}
+\end{align}
 
+Note that the accumulation commitment map $\beefycommitmap$ is set of pairs of indices of the output-yielding accumulated services to their accumulation result. This is utilized in equation \ref{eq:buildbeefymap}, when determining the accumulation-result tree root for the present block, useful for the \textsc{Beefy} protocol.
 
-We denote $R(s)$ the sequence of transfers received by a given service of index $s$, in order of them being sent from services of ascending index. (If some service $s$ received no transfers or simply does not exist then $R(s)$ would be validly defined as the empty sequence.) Formally:
+We have denoted the sequence of implied transfers as $\mathbf{t}$, ordered internally according to the source service's execution. We define a selection function $R$, which maps a sequence of deferred transfers and a desired destination service index into the sequence of transfers targeting said service, ordered primarily according to the source service index and secondarily their order within $\mathbf{t}$. Formally:
 \begin{align}
   R\colon \left\{\;\begin{aligned}
-    \N_S &\to \lseq\mathbb{T}\rseq \\
-    d &\mapsto \left[\,t \mid s \orderedin \mathbf{S},\ t \orderedin A(s)_\mathbf{t},\ t_d = d\,\right]
+    (\defxfers, \N_S) &\to \defxfers \\
+    d &\mapsto \left[\,t \mid s \orderedin \N_S,\ t \orderedin \mathbf{t},\ t_s = s,\ t_d = d\,\right]
   \end{aligned}\right.
 \end{align}
 
-The posterior state $\delta'$ may then be defined as the intermediate state with all the deferred effects of the transfers applied:
+The posterior state $\delta'$ may then be defined as the second intermediate state with all the deferred effects of the transfers applied:
 \begin{equation}
-  \delta' = \{ s \mapsto \Psi_T(\delta^\ddagger, a, R(a)) \mid (s \mapsto a) \in \delta^\ddagger \}
+  \delta' = \{ s \mapsto \Psi_T(\delta^\ddagger, a, R(\mathbf{t}, s)) \mid (s \mapsto a) \in \delta^\ddagger \}
 \end{equation}
 
 Note that $\Psi_T$ is defined in appendix \ref{sec:ontransferinvocation} such that it results in $\delta^\ddagger[d]$, \ie no difference to the account's intermediate state, if $R(d) = []$, \ie said account received no transfers.
+
+We define the final state of the ready queue and the accumulated map by integrating those work-reports which were accumulated in this block and shifting any from the prior state with the oldest such items being dropped entirely:
+\begin{align}
+  \accumulated'_{\mathsf{E} - 1} &= \srmap(\mathbf{W}^*_{\dots n}) \\
+  \forall i \in \N_\mathsf{E - 1}: \accumulated'_{i} &\equiv \accumulated_{i + 1} \\
+  \forall i \in \N_\mathsf{E} : \ready'^\circlearrowleft_{m - i} &\equiv \begin{cases}
+    E(\mathbf{W}^Q, \mathbf{W}^*_{\dots n}) &\when i = 0 \\
+    [] &\when 1 \le i < s \\
+    E(\ready^\circlearrowleft_{m - i}, \mathbf{W}^*_{\dots n}) &\when i \ge s\\
+    \multicolumn{2}{l}{\quad\where s = \min(\tau' - \tau, \mathsf{E})}
```

Comment:
The min(, E) here doesn't seem necessary? i is at most E-1 so the min won't affect any of the conditions.